values.yaml

```diff
@@ -515,6 +515,11 @@ jobservice:
   notification:
     webhook_job_max_retry: 3
     webhook_job_http_client_timeout: 3 # in seconds
+  reaper:
```

Comment:
It is more like a env variable that have not been exposed to the harbor.yml 
https://github.com/goharbor/harbor/blob/main/make/harbor.yml.tmpl
Do we need it configurable in harbor-helm?

Reply:
It is ok to keep it as it was

Reply:
it is configured in the `harbor/common/config/jobservice/config.yml`. If you deploy a Harbor v2.8.0, you will find it there. Therefore, we add it in `jobservice-cm.yaml`, instead of `jobservice-cm-env.yaml`.